app/src/main/java/edu/uw/myuw_android/LoginActivity.kt

```diff
@@ -42,11 +31,11 @@ class LoginActivity: AppCompatActivity() {
         authorizationService = AuthorizationService(this)
 
         if (UserInfoStore.readAuthState(this).isAuthorized) {
-            findViewById<TextView>(R.id.textView).text = "You are Authorized"
+            findViewById<TextView>(R.id.textView).text = "You are signed in"
```

Comment:
textView.text = ...
We have two TextViews both with id "textView". The other one is in activity_search.

Reply:
And also it's not good style to hardcode the text. Should be put in resources.